package.json

```diff
@@ -4,41 +4,40 @@
   "private": false,
   "description": "Aries Key Guardian for Kiva Protocol",
   "license": "Apache-2.0",
-  "type": "commonjs",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/kiva/protocol.git"
   },
   "scripts": {
-    "build": "rm -rf dist/ && tsc",
-    "format": "prettier --write \"src/**/*.ts\"",
-    "start": "rm -rf dist/ && tsc && node -r dotenv/config dist/main.js",
+    "build": "rm -rf dist/ && tsc && cp package.json dist/",
```

Comment:
Yeah, the difference is intentional. Aries-Controller is a library, which means it has exports and other people pull it in as a dependency. I changed the libraries to provide shallow imports (the `index.ts/js` file), so to not confuse Typescript projects, I wanted to also only have one declaration file, which is the whole `build:dts` thing. Because Aries-Key-Guardian is a service, it doesn't have any of those issues and we can just leave its declaration files alone.